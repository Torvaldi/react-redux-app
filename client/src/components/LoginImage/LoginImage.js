import React from 'react';
import propTypes from 'prop-types';
import './loginImage.css';


const LoginImage = (props) => {
  return (
      <img src={props.image} alt="login_ilustration" className="imageLogin" />
  );
}

LoginImage.propTypes = {
  image: propTypes.string.isRequired,
}


export default LoginImage;
