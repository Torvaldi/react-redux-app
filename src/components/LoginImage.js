import React from 'react';

import '../css/loginImage.css';
import image from '../img/login.jpg';

const LoginImage = (props) => {
  return (
      <img src={image} alt="image" className="imageLogin" />
  );
}

export default LoginImage;
