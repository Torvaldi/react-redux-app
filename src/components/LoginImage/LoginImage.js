import React from 'react';

import './loginImage.css';
import image from './login.jpg';

const LoginImage = (props) => {
  return (
      <img src={image} alt="image" className="imageLogin" />
  );
}

export default LoginImage;
