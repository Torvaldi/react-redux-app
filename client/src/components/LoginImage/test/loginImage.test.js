import React from 'react';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import image from '../../../asset/login.jpg';
import LoginImage from '../LoginImage';

configure({ adapter: new Adapter() });

describe('<LoginImage />', () => {
    // snapshot
    it('should render LoginImage Componenent', () => {
        const wrapper = shallow(<LoginImage image={image} />);
        expect(wrapper).toMatchSnapshot();
    });
    
    it('should render with props text', () => {
        const LoginImageComponent = mount(<LoginImage image={image} />);
        expect(LoginImageComponent.prop('image')).toEqual(image); 
    });
    
});