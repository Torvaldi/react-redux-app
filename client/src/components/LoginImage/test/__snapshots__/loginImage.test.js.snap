// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginImage /> should render LoginImage Componenent 1`] = `ShallowWrapper {}`;
